backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -0,0 +1,23 @@
+package com.crm.model.db;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity(name = "car_entity")
+public class CarEntity {
+
+    @Id
+    private int id;
+
+    private String vin;
+    private String registrationNumber;
+    private String brand;
+    private String model;
+    private int productionYear;
+    private int mileage;
+
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
Should be added @Column(name = type_id) or @JoinColumn(name = type_id). Not sure which is appropriate in this case.

Reply:
I think I solved this. Look at second commit.

Reply:
In db there is a column "type_id". Not sure if this will work if we use different name in @JoinColumn annotation?